src/app/submission/sections/form/section-form.component.spec.ts

```diff
@@ -44,6 +44,7 @@ import { SubmissionObjectDataService } from '../../../core/submission/submission
 import { ObjectCacheService } from '../../../core/cache/object-cache.service';
 import { RequestService } from '../../../core/data/request.service';
 import { createSuccessfulRemoteDataObject$ } from '../../../shared/remote-data.utils';
+import { cold } from 'jasmine-marbles';
```

Comment:
Note that you no longer need jasmine-marbles. Rxjs has marble support built in nowadays: https://rxjs.dev/guide/testing/marble-testing